### PR TITLE
mgr/volumes: cleanup leftovers from earlier purge job implementation

### PR DESCRIPTION
**Pull request by vshankar**

### CODE REVIEW

File: src/pybind/mgr/volumes/module.py [hunk 1]
```diff
@@ -1,27 +1,12 @@
 from threading import Event
```
Comment (ajarr) [inline src/pybind/mgr/volumes/module.py]: Remove this

Comment (vshankar) [reply src/pybind/mgr/volumes/module.py]: done

**Comment by batrick** [comment]
@joscollin do QA testing for this please.

**Comment by batrick** [comment]
jenkins test docs

**Pull request merged**